ci_framework/roles/libvirt_manager/tasks/attach_interface.yml

```diff
@@ -58,18 +58,13 @@
         mac_address: "{{ _lm_mac_address }}"
       ansible.builtin.import_tasks: reserve_ip.yml
 
+    # Note: This task will fail if the virt-net name is greater than 15 chars
```

Comment:
Can we add a conditional here to check the virt-net name should be <= 15 otherwise it will fail with a message.

Reply:
I did think on those lines but found the error message from `command` had the required information hence avoided it. Since we are failing at this point avoided the check. 

```
"stderr": "error: Failed to attach interface\nerror: Network interface name 'cifmw-net_osp_trunk' is too long: Numerical result out of range"
```

Let me know if you still feel having a check instead of the comment makes sense. We can add it.

Reply:
@psathyan Thank you for the detailed explanation. I think we can add the check. It will be useful for end user.

Reply:
Thank you.

The check is now available.